setup.py

```diff
@@ -579,6 +586,25 @@ def get_common_options(build_ext):
     LIBRARY_DIRS = []
     LIBRARIES = []
 
+    cpu_operation = os.environ.get('HOROVOD_CPU_OPERATIONS')
+    if cpu_operation:
+        print('INFO: Set default cpu operation to ' + cpu_operation)
+        MACROS += [('HOROVOD_CPU_OPERATIONS', "'%s'" % cpu_operation)]
```

Comment:
`HOROVOD_CPU_OPERATIONS_DEFAULT` or `HOROVOD_DEFAULT_CPU_OPERATIONS`?  Make sure to align with operations.cc.

Reply:
Sorry. A mistake.